scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -2267,7 +2267,6 @@ def _init_compat2binding(self) -> None:
                 _err(
                         f"'{binding_path}' appears in binding directories "
                         f"but isn't valid YAML: {e}")
-                continue
```

Comment:
just curious how did you find this? linter or eyeballs?

Reply:
CI failure